app/models/post.rb

```diff
@@ -5,4 +5,16 @@ class Post < ApplicationRecord
 
   validates :title, presence: true
   validates :text, presence: true
+
+  def update_user_posts_counter
+    author.update(posts_counter: author.posts.count)
+  end
+
+  def five_most_recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  def update_likes_counter
+    update(likes_counter: likes.count)
+  end
```

Comment:
- [ ] Kindly remove the update_likes_counter method from `post.rb` as you have already it to your likes model 👍🏿